scripts/zones/Misareaux_Coast/npcs/qm7.lua

```diff
@@ -0,0 +1,53 @@
+-----------------------------------
```

Comment:
If all the qms are the same code can make a global for the zone and just have each script pull the data from it.